ext/gettext/gettext.stub.php

```diff
@@ -13,18 +13,18 @@ function dgettext(string $domain, string $msgid): string {}
 
 function dcgettext(string $domain, string $msgid, int $category): string {}
 
-function bindtextdomain(string $domain, string $dir): string|false {}
+function bindtextdomain(string $domain, string $directory): string|false {}
 
 #ifdef HAVE_NGETTEXT
-function ngettext(string $msgid1, string $msgid2, int $n): string {}
+function ngettext(string $msgid_singular, string $msgid_plural, int $count): string {}
```

Comment:
I left `$msgid` alone, because it's used in the gettext file itself... However, I'd be happy to rename them to `$message_id` if that makes sense. I'm also not opposed to stay with `$msgid1` and `$msgid2`.

Reply:
It looks like the .po file itself calls these `$msgid` and `$msgid_plural`. I have no familiarity with this ext and no strong opinion on naming here.

Reply:
Unfortunately, I am not very happy with these names, they look ugly. So I've just had a look at what param names Python opted for: https://docs.python.org/3/library/gettext.html and I find them much-much better. What do you think?

Reply:
Looks reasonable :)

Reply:
Done